docs/how-to-guides/javascript/README.md

```diff
@@ -1,6 +1,6 @@
-# Getting Started with JavaScript
+# How to use JavaScript with Gutenberg 
```

Comment:
I looked around the docs and "Block Editor" is used more frequently.

Not a blocker for this PR.

Reply:
I was conflicted. I take your comment to as typing the scale :-)  "How to use JavaScript with the Block Editor"